CorinneTest/IPerson.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
```

Comment:
are you actually using these interfaces?

Reply:
not necessary for these projects - deleting IPerson and IPersonFactory interfaces